modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -429,6 +429,196 @@ def dt_month(self):
             self._modin_frame.dt_extract("month"), self._shape_hint
         )
 
+    def dt_week(self):
+        """Extract week from Datetime info
```

Comment:
I know you follow docs style from other methods in this class, but there was an agreement on it and we should use new formatting in the new code. You just need to use something like this:
```
Returns
-------
DFAlgQueryCompiler
    A new query compiler.
```